client/src/+assets/details.tsx

```diff
@@ -219,179 +223,341 @@ export const AssetDetails: NextPage = () => {
 
   return (
     <>
-
       <UiLayout type="container">
-        <UiText wrapper="h1" type="h1" variants={['heading']}>
-          Details
-        </UiText>
-        <UiText type="h2" wrapper="h2" variants={['heading']}>
-          {metadata.main.name}
-        </UiText>
-
         <UiLayout align="start" type="sides">
           <div className={b('content')}>
-            <UiText type="h3" wrapper="h3" variants={['underline']}>
-              Description
+            <UiText type="h2" wrapper="h2">
+              Report
```

Comment:
I don't know what field to map into this topmost header 🤷
@ialberquilla @josepablofm78 do you know what exactly should be there? [Figma](https://www.figma.com/file/qbvI8d1LQ353WlzUf0Y5Hp/Nevermined?node-id=3260%3A11320) shows this:
<img width="1009" alt="Screenshot 2022-09-23 at 16 10 29" src="https://user-images.githubusercontent.com/1643399/191993820-d33c17c4-6c65-4b0c-b258-9aa91b229d00.png">